src/db/methods/collections/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""This module for working with collections"""
+
+from . collections import users, \
+                            roles, \
+                            domains, \
+                            contests, \
+                            group_roles, \
+                            group_members, \
+                            internal_counters
```

Comment:
Это же вообще не работает, какой `from . collections`
```suggestion
from .collections import (
    users, roles, domains, contests, group_roles, group_members, internal_counters
)
```

Reply:
Ну либо также как внизу, все через новую строку.